diff --git a/source/scripts/init/system/utopia_init.sh b/source/scripts/init/system/utopia_init.sh
index 8b6533b..0450c36 100755
--- a/source/scripts/init/system/utopia_init.sh
+++ b/source/scripts/init/system/utopia_init.sh
@@ -441,14 +441,14 @@ wan_ifname=`sysevent get wan_ifname`
 
 #disable telnet / ssh ports
 iptables -A INPUT -i "$lan_ifname" -p tcp --dport 23 -j DROP
-iptables -A INPUT -i "$lan_ifname" -p tcp --dport 22 -j DROP
+#iptables -A INPUT -i "$lan_ifname" -p tcp --dport 22 -j DROP
 iptables -A INPUT -i "$cmdiag_ifname" -p tcp --dport 23 -j DROP
-iptables -A INPUT -i "$cmdiag_ifname" -p tcp --dport 22 -j DROP
+#iptables -A INPUT -i "$cmdiag_ifname" -p tcp --dport 22 -j DROP
 
 ip6tables -A INPUT -i "$lan_ifname" -p tcp --dport 23 -j DROP
-ip6tables -A INPUT -i "$lan_ifname" -p tcp --dport 22 -j DROP
+#ip6tables -A INPUT -i "$lan_ifname" -p tcp --dport 22 -j DROP
 ip6tables -A INPUT -i "$cmdiag_ifname" -p tcp --dport 23 -j DROP
-ip6tables -A INPUT -i "$cmdiag_ifname" -p tcp --dport 22 -j DROP
+#ip6tables -A INPUT -i "$cmdiag_ifname" -p tcp --dport 22 -j DROP
 
 #protect from IPv6 NS flooding
 ip6tables -t mangle -A PREROUTING -i "$ecm_wan_ifname" -d ff00::/8 -p ipv6-icmp -m icmp6 --icmpv6-type 135 -j DROP
@@ -698,7 +698,7 @@ syscfg set ntp_status 2
 echo_t "[utopia][init] setting Multicast MAC before any switch configs"
 $UTOPIA_PATH/service_multinet_exec set_multicast_mac &
 
-if [ "$MODEL_NUM" = "DPC3939B" ] || [ "$MODEL_NUM" = "DPC3941B" ]; then
+if [ "$MODEL_NUM" = "DPC3939B" ] || [ "$MODEL_NUM" = "DPC3941B" ] || [ "$MODEL_NAME" = "Filogic" ]; then
 	echo_t "[utopia][init] started dropbear process"
 	/etc/utopia/service.d/service_sshd.sh sshd-start &
 fi
-- 
2.18.0

