diff --git a/interfaces/opensync.ovsschema b/interfaces/opensync.ovsschema
index 0891bd1..254fceb 100644
--- a/interfaces/opensync.ovsschema
+++ b/interfaces/opensync.ovsschema
@@ -6399,6 +6399,91 @@
       },
       "isRoot": true
     },
+    "Wifi_Blaster_Config": {
+      "columns": {
+        "plan_id": {
+          "type": {
+            "key": {
+              "type": "string",
+              "maxLength": 37
+            }
+          },
+          "mutable": false
+        },
+        "step_id_and_dest": {
+          "type": {
+            "key": {
+                "type": "string",
+                "maxLength": 11
+            },
+            "value": {
+                "type": "string",
+                "maxLength": 18
+            },
+            "min": 0,
+            "max": 64
+          },
+          "mutable": false
+        },
+        "blast_packet_size": {
+          "type": "integer",
+          "mutable": false
+        },
+        "blast_duration": {
+          "type": "integer",
+          "mutable": false
+        },
+        "blast_sample_count": {
+          "type": "integer",
+          "mutable": false
+        },
+        "blast_config": {
+          "type": {
+            "key": {
+              "type": "string",
+              "maxLength": 65
+            },
+            "value": {
+              "type": "string",
+              "maxLength": 65
+            },
+            "min": 0,
+            "max": 64
+          },
+          "mutable": false
+        }
+      },
+      "isRoot": true
+    },
+    "Wifi_Blaster_State": {
+      "columns": {
+        "plan_id": {
+          "type": {
+            "key": {
+              "type": "string",
+              "maxLength": 65
+            }
+          },
+          "mutable": false
+        },
+        "state": {
+          "type": {
+            "key": {
+              "type": "string",
+              "enum": [
+                "set",
+                [
+                  "new",
+                  "complete"
+                ]
+              ]
+            }
+          },
+          "mutable": false
+        }
+      },
+      "isRoot": true
+    },
     "Band_Steering_Config": {
         "columns": {
             "if_name_2g": {

